content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -15,43 +15,35 @@ Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites)
 
 ## Installation steps
 
-1. If using a Helm version prior to 2.10.0, install Istio's [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
-via `kubectl apply`, and wait a few seconds for the CRDs to be committed to the Kubernetes API server:
+You have two mutually exclusive options to install Istio:
+- To use Kubernetes manifests to deploy Istio,  follow the instructions for [option #1](#option1).
+- To use [Helm's Tiller pod](https://helm.sh/) to manage your Istio release, follow the instructions for [option #2](#option2).
 
-    {{< text bash >}}
-    $ kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml
-    {{< /text >}}
+### Option 1: Install with Helm via `helm template` {#option1}
+
+Choose this option if your cluster doesn't have [Tiller](https://github.com/kubernetes/helm/blob/master/docs/architecture.md#components) deployed and you don't want to install it.
 
-1. Choose one of the following two
-**mutually exclusive** options described below.
+1. Install all the Istio's [Custom Resource Definitions or CRDs for short](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the Kube api-server:
 
-### Option 1: Install with Helm via `helm template`
+    {{< text bash >}}
+    $ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
+    {{< /text >}}
 
 1. Render Istio's core components to a Kubernetes manifest called `istio-minimal.yaml`:
 
     {{< text bash >}}
+    $ cat @install/kubernetes/namespace.yaml@ > $HOME/istio-minimal.yaml
     $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
-      --set security.enabled=false \
-      --set ingress.enabled=false \
-      --set gateways.istio-ingressgateway.enabled=false \
-      --set gateways.istio-egressgateway.enabled=false \
-      --set galley.enabled=false \
-      --set sidecarInjectorWebhook.enabled=false \
-      --set mixer.policy.enabled=false \
-      --set mixer.telemetry.enabled=false \
-      --set prometheus.enabled=false \
-      --set global.proxy.envoyStatsd.enabled=false \
-      --set pilot.sidecar=false > $HOME/istio-minimal.yaml
+      --values install/kubernetes/helm/istio/values-istio-minimal.yaml >> $HOME/istio-minimal.yaml
     {{< /text >}}
 
 1. Install the Pilot component via the manifest:
 
     {{< text bash >}}
-    $ kubectl create namespace istio-system
     $ kubectl apply -f $HOME/istio-minimal.yaml
     {{< /text >}}
 
-### Option 2: Install with Helm and Tiller via `helm install`
+### Option 2: Install with Helm and Tiller via `helm install` {#option2}
```

Comment:
not sure what #option1 and #option2 do.  The links work without it, do they not?

Reply:
They do if you include the entire heading on the link. Adding the label `{#option2}` makes the link easier to maintain because the heading can change without breaking the link.